deploy/osd-21066-camo-olm-dance/01-cronjob.yaml

```diff
@@ -0,0 +1,43 @@
+---
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: sre-operator-reinstall
+  namespace: openshift-monitoring
+spec:
+  ttlSecondsAfterFinished: 100
+  failedJobsHistoryLimit: 1
+  successfulJobsHistoryLimit: 3
+  concurrencyPolicy: Replace
+  schedule: "*/5 * * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          serviceAccountName: sre-operator-reinstall-sa
+          restartPolicy: Never
```

Comment:
why do we go for no restart here? :thinking:

Reply:
ah i guess the cronjob will run again and it does not matter

Reply:
I copy pasted most of this from the previous attempt to olm dance, but I believe `restartPolicy`  would restart the pod for a job in case the pod fails, but we would have a new job after 5 minutes anyway, so I think it's no big deal.